### PR TITLE
Add combined file description in README.md

### PR DESCRIPTION
**Pull request by ashutosh-ukey**
Changes:

- add description in README.md for crypto-payments.{json,yaml} and circle-apis.{json,yaml}
- remove changelog link (file already present in root folder)
- refactor OpenAPI version location in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,15 @@
...
-* `account-doc.{json,yaml}:` OpenAPI 3.0 spec for APIs to manage your account balance.
-* `accounts-doc.{json,yaml}:` OpenAPI 3.0 spec for APIs to manage custodial wallets.
-* `general-doc.{json,yaml}:` OpenAPI 3.0 spec for common endpoints shared across all Circle APIs.
-* `payments-doc.{json,yaml}:` OpenAPI 3.0 spec for APIs to accept payments.
-* `payouts-doc.{json,yaml}:` OpenAPI 3.0 spec for APIs to make payouts.
+* `account.{json,yaml}:` APIs to manage your account balance.
+* `accounts.{json,yaml}:` APIs to manage custodial wallets.
+* `circle-apis.{json,yaml}:` all Circle APIs combined into one file.
+* `crypto-payments.{json,yaml}:` Crypto Payments collection Circle APIs.
+* `general.{json,yaml}:` common endpoints shared across all Circle APIs.
```
Comment (prakashb-circle) [inline README.md:11]: _**Common**_ instead of **_common_**

File: README.md [hunk 2]
```diff
@@ -2,16 +2,15 @@
...
-Files can be found in the `openapi/{json or yaml}` directory:
-
-* `account-doc.{json,yaml}:` OpenAPI 3.0 spec for APIs to manage your account balance.
-* `accounts-doc.{json,yaml}:` OpenAPI 3.0 spec for APIs to manage custodial wallets.
-* `general-doc.{json,yaml}:` OpenAPI 3.0 spec for common endpoints shared across all Circle APIs.
-* `payments-doc.{json,yaml}:` OpenAPI 3.0 spec for APIs to accept payments.
-* `payouts-doc.{json,yaml}:` OpenAPI 3.0 spec for APIs to make payouts.
+* `account.{json,yaml}:` APIs to manage your account balance.
+* `accounts.{json,yaml}:` APIs to manage custodial wallets.
+* `circle-apis.{json,yaml}:` all Circle APIs combined into one file.
```
Comment (prakashb-circle) [inline README.md:9]: **_All_** instead of **_all_**

File: README.md (hunk 1 above)
Comment (ashutosh-ukey) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (ashutosh-ukey) [reply README.md]: Fixed

**Review by prakashb-circle — APPROVED** [review]

**Pull request merged**